### PR TITLE
Fix: Sort out .travis.yml

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] sorts out `.travis.yml`

💁 For reference, see https://docs.travis-ci.com/user/customizing-the-build/#The-Build-Lifecycle.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-  - composer self-update
-  - composer install --prefer-source --no-interaction --dev
-script: 'vendor/bin/phpspec run'
+
 php:
   - 5.5
   - 5.6
+
+before_install:
+  - composer self-update
```
Comment (localheinz) [inline .travis.yml]: It makes more sense to update composer itself in a `before_install`.

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+
+install:
+  - composer install --prefer-source --no-interaction --dev
+
+script:
+  - vendor/bin/phpspec run
```
Comment (localheinz) [inline .travis.yml]: It makes more sense to allow listing commands here.

**Comment by thommahoney** [comment]
Cool thanks @localheinz

**Comment by alieander** [comment]
👍

**Pull request merged**